CMakeLists.txt

```diff
@@ -111,11 +111,14 @@ if (NOT DEFINED QL_BOOST_VERSION)
 endif()
 
 if (NOT DEFINED QL_BOOST_COMPONENTS)
-    # Boost thread and unit_test_framework needed for tests and benchmark
+    set(QL_BOOST_COMPONENTS "")
+    # Boost unit_test_framework needed for tests and benchmark
     if (QL_BUILD_TEST_SUITE OR QL_BUILD_BENCHMARK)
-        set(QL_BOOST_COMPONENTS thread unit_test_framework)
-    else()
-        set(QL_BOOST_COMPONENTS thread)
+        set(QL_BOOST_COMPONENTS "${QL_BOOST_COMPONENTS};unit_test_framework")
```

Comment:
Is colon-separated the same as space-separated? Just curious.

Reply:
Yes, both are acceptable formats for specifying a list. See [here](https://cmake.org/cmake/help/latest/command/set.html) and [here](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#cmake-language-lists).

However sometimes CMake has problems unpacking space-separated lists, as was the case here, so using a semicolon-separated list is the way to go.